README.md

```diff
@@ -10,9 +10,12 @@ php-coveralls
 
 PHP client library for [Coveralls](https://coveralls.io). 
 
-# API doc
+# Future deprecations
 
-[API doc](http://satooshi.github.io/php-coveralls/api/) is generated by [ApiGen](http://apigen.org/)
+We will make a breaking change regarding to the config file format on
```

Comment:
Ah OK. Do you think it's worthwhile to add some code to give deprecation warnings?